api/src/main/resources/workbench.yaml

```diff
@@ -1195,6 +1210,8 @@ definitions:
         type: "array"
         items:
           $ref: "#/definitions/Authority"
+      idVerificationStatus:
```

Comment:
How is this going to get used on the client side?

Should we create an enum or something for this?

Reply:
Enum was a good call.